packages/sheets-ui/src/controllers/menu/merge.menu.ts

```diff
@@ -39,7 +48,7 @@ export function CellMergeMenuItemFactory(accessor: IAccessor): IMenuSelectorItem
         type: MenuItemType.SUBITEMS,
         // selections: [...MERGE_CHILDREN],
         hidden$: getMenuHiddenObservable(accessor, UniverInstanceType.SHEET),
-        disabled$,
+        disabled$: disableByPermissionAndSelections$,
```

Comment:
You don't need to manually call `next` on `disableByPermissionAndSelections$`.

```suggestion
        disabled$: disabled$.pipe(
        	combineLatestWith(selectionsHasCross$),
        	map(([disable, hasCross]) => disable || hasCross)
    	)
```

is sufficient.